### PR TITLE
Relative paths

### LINKED ISSUE #65: Have relative paths in results object
```gherkin
As a user
I want Mugshot to give me relative paths
So I can easily use the result object in another environments
```

### PR DESCRIPTION
**Pull request by adela-istrate-hs**
Closes #65

### CODE REVIEW

**Comment by adela-istrate-hs** [comment]
@valentin-radulescu-hs @andrei-picus-hs Is this ok? Can I merge?

**Comment by valentin-radulescu-hs** [comment]
LGTM

File: tests/integration/test.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ var fs = require('fs');
 
 var name = 'great',
     ext = '.png',
-    dir = path.join(__dirname, '..', '..', 'visual-tests'),
+    dir = path.join('visual-tests'),
```
Comment (andrei-picus-hs) [inline tests/integration/test.js]: No need for `join` here.

**Comment by andrei-picus-hs** [comment]
LGTM except https://github.com/uberVU/mugshot/pull/66#discussion_r77483805.

**Pull request merged**